static/css/components/header-bar.less

```diff
@@ -145,17 +145,17 @@
       height: 44px;
       padding: 0 3px;
     }
+  }
 
-    .hamburger-component__checkbox {
-      opacity: 0;
-      
-      & ~ .hamburger-dropdown-component ul {
-        display: none;
-      }
+  .hamburger-component__checkbox {
```

Comment:
This is embedded inside the selector started on L131, move it out, to squash the error.

Reply:
I think this is outside that selector already. The whole snippet looks like this:
```
.hamburger-component {                            // Line 131
    -webkit-box-ordinal-group: 1;
    -ms-flex-order: 0;
    order: 0;
    margin-right: 5px;

    label {
      background: transparent;
      border: none;
      cursor: pointer;
    }

    img {
      width: 44px;
      height: 44px;
      padding: 0 3px;
    }
}

  .hamburger-component__checkbox {
    opacity: 0;
    
    & ~ .hamburger-dropdown-component ul {
      display: none;
    }

    &:checked ~ .hamburger-dropdown-component ul {
      display: block;
    }
  }

  .logo-component {
    -webkit-box-flex: 1;
    -ms-flex: 1;
    flex: 1;
    -webkit-box-ordinal-group: 2;
    -ms-flex-order: 1;
    order: 1;
    .......
```

Reply:
Note the leading tabs at the start of the line. This file is pretty big which makes it hard to read (should probably be split up into multiple components at some later date.

This rule is inside the header-bar started on L13. The header-bar selector is making this unnecessarily specific.

.header-bar .hamburger-component__checkbox ~ .hamburger-dropdown-component ul

Try moving it to just above the `.account-component` selector on